AUTHORS.rst

```diff
@@ -10,6 +10,6 @@
 Authors
 =======
 
-- Lars Nielsen
-- Pablo Panero
 - Guillaume Viger
+- Lars Holm Nielsen
+- Pablo Panero
```

Comment:
Was I right in thinking this change is to have authors in alphabetical order of first name?

Reply:
Correct, we do alphabetical sorting of first name (so you can use the editor to quickly sort lines) and that there's no discussion on the order of names.